tox.ini

```diff
@@ -9,6 +9,7 @@ commands = {posargs:coverage run -m pytest}
 
 
 [testenv:flake8]
+basepython = python3.5
```

Comment:
Out of interest, why 3.5 here?

Reply:
Totally arbitrary. I think just because 3.5 was the original supported version?

Reply:
Fair enough!